libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/guidance/junction/view/MapboxJunctionView.kt

```diff
@@ -45,14 +45,14 @@ class MapboxJunctionView : AppCompatImageView {
      *
      * @param result type [Expected] which holds either the junction or error
      */
-    fun render(result: Expected<JunctionValue, JunctionError>) {
-        when (result) {
-            is Expected.Success -> {
-                setImageBitmap(result.value.bitmap)
-            }
-            is Expected.Failure -> {
+    fun render(result: Expected<JunctionError, JunctionValue>) {
+        result.fold(
+            { // error
```

Comment:
```suggestion
            { error ->
```

Reply:
This results in an unused variable lint warning.

Reply:
😥